ghcide/ghcide.cabal

```diff
@@ -269,7 +264,7 @@ executable ghcide
   hs-source-dirs:     exe
   ghc-options:
     -threaded -Wall -Wincomplete-uni-patterns -Wno-name-shadowing
-    -rtsopts "-with-rtsopts=-I0 -A128M -T"
+    -rtsopts "-with-rtsopts=-I0 -A128M -T" -Wunused-packages
```

Comment:
Tiny nitpick, can we move `-Wunused-packages` to the line above, then the warnings are nicely grouped

Reply:
Applied